index.js

```diff
@@ -60,7 +60,7 @@ module.exports = function runServer(isDist, middlewareFn) {
     });
     startListening();
   } else if (env.HOST || process.env.HOST) {
-    util.ngServe(env.HOST || process.env.HOST, function serveDev(ngServePort) {
+    util.ngServe(env.HOST || process.env.HOST, isDist, function serveDev(ngServePort) {
```

Comment:
Isn't this redundant?  `ngServe` is only used in dev.  And this will always be false, per the above `if (isDist)`.